src/daemon/dlt-daemon.c

```diff
@@ -957,6 +949,32 @@ int dlt_daemon_local_init_p2(DltDaemon *daemon, DltDaemonLocal *daemon_local, in
     return 0;
 }
 
+int dlt_daemon_local_init_p3(DltDaemon *daemon, DltDaemonLocal *daemon_local, int verbose)
+{
+    PRINT_FUNCTION_VERBOSE(verbose);
+
+    if ((daemon == 0) || (daemon_local == 0)) {
+        dlt_log(LOG_ERR, "Invalid function parameters used for function dlt_daemon_local_init_p3()\n");
+        return -1;
+    }
+
+    /* init offline trace */
+    if (((daemon->mode == DLT_USER_MODE_INTERNAL) || (daemon->mode == DLT_USER_MODE_BOTH)) &&
+        daemon_local->flags.offlineTraceDirectory[0]) {
+        if (dlt_offline_trace_init(&(daemon_local->offlineTrace),
+                                   daemon_local->flags.offlineTraceDirectory,
+                                   daemon_local->flags.offlineTraceFileSize,
+                                   daemon_local->flags.offlineTraceMaxSize,
+                                   daemon_local->flags.offlineTraceFilenameTimestampBased) == -1) {
+            dlt_log(LOG_ERR, "Could not initialize offline trace\n");
+            return -1;
+        }
+    }
+
+    return 0;
+}
+
+
```

Comment:
Extra line. I propose to run uncrustify to your modified code, as it's written in README.